tallycash/extension
# Add `checksum` validation

## Description
- This PR adds `checksum` validation when user enter address on `send` screen
- Purpose is to prevent users from sending funds to the wrong address
- Validation logic is provided by `@tallyho/hd-keyring` library which is already in use

## Demo 

<img width="381" alt="Screenshot 2022-10-31 at 7 47 49 PM" src="https://user-images.githubusercontent.com/104683677/199040570-1d50898e-308d-46eb-9503-ad8f370077fc.png">


## Type of Change
- [X] Address validation

## Testing information
- Activate `SUPPORT_RSK` env flag
- Go to send screen
- Enter the address according to network
- It should validate checksum according to network

## Checklist
- [X] Git hooks enabled
- [X] Git commit is signed


Latest build: [extension-builds-2526](https://github.com/tallycash/extension/suites/9127807837/artifacts/423937094) (as of Fri, 04 Nov 2022 10:34:49 GMT).
